test/cli/saveSequence.js

```diff
@@ -0,0 +1,38 @@
+require('../../src/ImageSequencer');
+sequencer = ImageSequencer({ ui: true });
+const saveSequence = require('../../src/cli/saveSequence.js');
+const test = require('tape');
+const { Command } = require('commander');
+
+
+test('testing save sequence function', function (t) {
+  try {
+    let program = new Command();
+    program
+      .option('--save-sequence [string]', 'Name space separated with Stringified sequence');
+
+    program.parse(['node', 'test', '--save-sequence', '"invert brightness"']);
+
+    if (program.saveSequence)
```

Comment:
What is program.saveSequence supposed to be?

Reply:
It's checking if `--saveSequence` was parsed or not. 

It is also done in index.js 
https://github.com/publiclab/image-sequencer/blob/14f13c300bf8ed360ea142322e00f02371ca9b49/index.js#L29